agot-bg-website/chat/consumers.py

```diff
@@ -98,6 +103,28 @@ async def receive_json(self, data):
                     'created_at': message.created_at.isoformat()
                 }
             )
+
+            try:
+                game = await database_sync_to_async(lambda: Game.objects.get(id=data['gameId']))()
+                pbem_active = game.view_of_game.get("settings", False) and game.view_of_game.get("settings").get("pbem", False) == True
+                if not self.room.public and pbem_active:
+                    # notify the other user about a new private message
+                    other_user = await database_sync_to_async(lambda: self.room.users.prefetch_related('user').filter(~Q(id=user.id)).first())()
+                    if other_user is not None:
+                        mailBody = render_to_string('agotboardgame_main/new_private_message.html',
+                            {'message': message.text, 'receiver': other_user, 'sender': user, 'game': game,
+                            'game_url': f'https://swordsandravens.net/play/{game.id}' })
+                        print (mailBody)
+                        mail = [(
+                            f'You received a new private message in game: \'{game.name}\'',
+                            mailBody, DEFAULT_FROM_MAIL, [other_user.user.email]
+                        )]
+
+                        print("before e-mail-send")
+                        send_mass_mail(mail)
```

Comment:
@Longwelwind I managed to make it work until here. But then I receive `Connection unexpectly closed`. Do you have any idea what needs to be done?